test/integration/spec/rtcpeerconnection.js

```diff
@@ -36,19 +36,17 @@ describe('RTCPeerConnection', function() {
     signalingStates.forEach(testAddIceCandidate);
   });
 
-  describe('#getLocalStreams, called from signaling state', () => {
-    signalingStates.forEach(testGetLocalStreams);
-  });
-
-  describe('#getRemoteStreams, called from signaling state', () => {
-    signalingStates.forEach(testGetRemoteStreams);
+  (isChrome ? describe : describe.skip)('#getSenders', () => {
+    signalingStates.forEach(testGetSenders);
   });
 
   describe('#close, called from signaling state', () => {
     signalingStates.forEach(testClose);
   });
 
-  (isSafari ? describe.skip : describe)('#addStream', testAddStream);
+  describe('#addTrack', testAddTrack);
+
+  (isChrome ? describe : describe.skip)('#removeTrack', testRemoveTrack);
```

Comment:
Why are we skipping for non-Chrome? EDIT: I guess because `removeTrack` isn't working in Firefox/Safari yet?

Reply:
@markandrus Yes. I've referred to the relevant bugs in the comments.